backend/internetarchive/internetarchive.go

```diff
@@ -226,7 +308,10 @@ func NewFs(ctx context.Context, name, root string, m configmap.Mapper) (fs.Fs, e
 	}
 	f.setRoot(root)
 	f.features = (&fs.Features{
-		BucketBased: true,
+		BucketBased:   true,
+		ReadMetadata:  true,
+		WriteMetadata: true,
```

Comment:
This should be false if we can't write metadata

Reply:
We can write/update it (like we do for setting mtime) but just not implemented

Reply:
So I was expecting some interface to add metadata, but fs.ObjectInfo in Update() has fs.Metadata implemented?

Reply:
Thanks, it's reflected